api.md

```diff
@@ -3265,7 +3265,7 @@ All sizes are in bytes.
 
 |Name|Description|
 |---|---|
-| `main_table` | (REGCLASS) Name of the hypertable |
+| `hypertable` | (REGCLASS) Hypertable to show stats for |
```

Comment:
```suggestion
| `hypertable` | Hypertable to show stats for. |
```

Reply:
I believe the "preferred" format is to include type argument's TYPE (we aren't great here, but I guess would like to trend towards better).